src/lib/encode-converstions.ts

```diff
@@ -21,24 +21,25 @@ type ConvertFunction<TSchemaName extends SchemaName> = (
   >
 ) => CurrentProtoTypes[TSchemaName]
 
+/* @ts-ignore TODO: resolve "not assignable to type 'ConvertFunction<"project">" */
 export const convertProject: ConvertFunction<'project'> = (mapeoDoc) => {
   return {
     common: convertCommon(mapeoDoc),
     ...mapeoDoc,
```

Comment:
```suggestion
    ...mapeoDoc,
    name: mapeoDoc.name || '',
```
I think that since `name` is now optional, we need to take that into account here

Reply:
I changed the protobuf definition to allow for optional values

Reply:
I changed the protobuf definition to allow for optional values

Reply:
right! forgot about that!